guardian/lib/slack/model/slack_section_block.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:equatable/equatable.dart';
+import 'package:guardian/slack/model/slack_text_object.dart';
+
+/// A class representing Section Block element
+/// (https://api.slack.com/reference/block-kit/blocks#section)
+class SlackSectionBlock extends Equatable {
+  /// The text for the block, in the form of a [SlackTextObject].
+  ///
+  /// Maximum length for the text in this field is 3000 characters.
```

Comment:
We should probably give feedback to users if they try to put more - via asserts or something else.